klasses/models.py

```diff
@@ -82,6 +85,35 @@ def payment_deadline(self):
         """
         return self.installment_set.aggregate(payment_deadline=models.Max('deadline'))['payment_deadline']
 
+    @property
+    def next_installment(self):
+        """
+        Get the next installment
+        """
+        return self.installment_set.filter(deadline__gte=datetime.datetime.now(tz=pytz.UTC)).first()
```

Comment:
The test already handles multiple installments, but it was working because they are created in the right order.